### PR TITLE
Fix uncaught ts issues

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

- makes pack-up stricter
- fix the issues appearing after the pack-up change
	- entity service - convert query params incompat
	- utils circular dep with database
	- build command
	- email plugin

### Why is it needed?

- To catch all the typing errors

### CODE REVIEW

File: packages/core/helper-plugin/package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "watch": "pack-up watch"
   },
   "dependencies": {
+    "@strapi/utils": "4.17.1",
```
Comment (joshuaellis) [inline packages/core/helper-plugin/package.json:53]: we only use the types, shouldn't this just be a `devDep`?

**Review by joshuaellis — COMMENTED** [review]
minor comment, but the changes look fine.

File: packages/core/helper-plugin/package.json (hunk 1 above)
Comment (alexandrebodin) [reply packages/core/helper-plugin/package.json:53]: sure 👍

**Review by Convly — APPROVED** [review]
LGTM, thanks!

**Review by joshuaellis — APPROVED** [review]
Great tidying 

I Love It Reaction GIF by Magic: The Gathering

**Review by innerdvations — APPROVED** [review]
LGTM, thanks!

**Pull request merged**